docs/components/List/List.stories.mdx

```diff
@@ -1,4 +1,4 @@
-import { Meta } from "@storybook/addon-docs";
+import { Meta, Canvas } from "@storybook/addon-docs";
```

Comment:
For anyone looking at this wondering why we had to edit this file. We're overriding Storybooks components in the new docs site. We need to actually import the components we're using though, in order for us to override.

This was causing an error on the new docs site. Not sure why it wasn't doing the same here. Maybe storybook provides those for free?